README.md

```diff
@@ -187,6 +187,40 @@ envsubst < values-dev-cluster-ogdc.yaml | helm upgrade --install ${{ env.CHART_N
 - `latest` - Latest development build from main branch
 - `0.1.0`, `0.2.0`, etc. - Specific release versions (created from git tags)
 
+
+## Versioning and Releases
+
+Version management and the creation of release tags are automated using **bump-my-version**.
+
+* **Documentation:** [https://callowayproject.github.io/bump-my-version/](https://callowayproject.github.io/bump-my-version/)
+
+Contributors should **not** manually update version numbers (e.g., in `helm/Chart.yaml`). All official version changes should be performed through this tool.
+
+### How to Create a New Release
+
+To perform a version bump and create a new release:
+
+1.  Ensure all changes for the release are outlined in the CHANGELOG with the `## NEXT_VERSION` header. 
+2.  Use the `bump-my-version` command followed by the part you wish to increment (`patch` or `minor`):
+    * To bump the **patch** version (e.g., 0.1.0 → 0.1.1):
+        ```bash
+        bump-my-version patch
+        ```
+    * To bump the **minor** version (e.g., 0.1.0 → 0.2.0):
+        ```bash
+        bump-my-version minor
+        ```
+3.  The command will:
+    * Update the version in all configured files (e.g., `helm/Chart.yaml`).
+    * Create a commit with the configured message.
```

Comment:
I think this and the next bullet need to be removed, and step 4/5 updated to reflect our workflow. The documentation should outline the expectation that contributors will manually commit the changes bumpversion makes and that a tag will be made and pushed to the main branch after changes are merged.